build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <SdkBrandName>Microsoft .NET Core 1.0.0 - SDK Preview 2</SdkBrandName>
```

Comment:
I think that's a bridge for when we get there. Worst case we probably build the localization into the filename and do conditional imports. It has to be possible in msbuild tho.